src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/CompareConverter.shared.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	/// <summary>
+	/// Converts an object that implements IComparable to an object or a boolean based on a comparaison.
+	/// </summary>
+	public class CompareConverter : CompareConverter<object>
+	{
+	}
+
+	/// <summary>
+	/// Converts an object that implements IComparable to an object or a boolean based on a comparaison.
+	/// </summary>
+	public class CompareConverter<TObject> : ValueConverterExtension, IValueConverter
```

Comment:
Since we allow inheritance from this class should this be abstract?

Reply:
Could you provide an example where inheritance is required? otherwise I think w can make it sealed as you suggested in your previous comment.

Reply:
you do it at line 11, so the dev can take that as inspiration and want to use some strongly typed type.